pbench_copy_data

```diff
@@ -75,7 +75,7 @@ if [[ "$controller" == *"--"* ]]; then
 	exit 1
 fi
 
-echo pbench-copy-results --user "\"$user\"" --prefix "\"$results_prefix\"" >> /tmp/debugging
+echo pbench-copy-results --user "\"$user\"" --controller "\"$results_prefix\"" >> /tmp/debugging
```

Comment:
Shouldn't this line be executed after the test below (line 79)?

Reply:
We want to log what is being executed before we execute it.  That way if  it hangs we know what we passed.

Reply:
Added --controller option for debug, and moved it to just before we execute the command.